tests/unit/express-checkout/test-class-wc-payments-express-checkout-button-display-handler.php

```diff
@@ -13,7 +13,7 @@
 use WCPay\WooPay\WooPay_Utilities;
 
 /**
- * WC_Payments_WooPay_Button_Handler_Test class.
```

Comment:
Fixed the class' description to reflect the class in the file.

Reply:
💾 + 🧴. 🤦